### PR TITLE
feat: Move utils from CredentialService [DEV-3051]

### PR DESCRIPTION
**Pull request by anikitinDSR**

### CODE REVIEW

**Comment by ankurdotb** [comment]
Task linked: [DEV-3051 Automatically create the customer account for users after initial login](https://app.clickup.com/t/6600954/DEV-3051)

File: src/utils.ts [hunk 1]
```diff
@@ -295,4 +300,14 @@ export function createMsgResourcePayloadToSign(payload: Partial<MsgCreateResourc
   return MsgCreateResourcePayload.encode(
     MsgCreateResourcePayload.fromPartial(payload)
   ).finish()
+}
+
+export function getCosmosAccount(kid: string): string {
```
Comment (DaevMithran) [inline src/utils.ts:305]: @anikitinDSR let's change this to publicKeyHex instead of kid.

**Review by DaevMithran — CHANGES_REQUESTED** [review]

File: src/utils.ts (hunk 1 above)
Comment (Eengineer1) [reply src/utils.ts:305]: Mostly agreed with Daev here.

Coincidentally, kid + publicKeyHex are identical by value, but that's not always the case with all key id references.

File: tests/constants.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// They are connected 
+export const PUBKEY_HEX = "04adf6cad45e58a7e1908bebefcc358de229c108fb1170566f83be5ce028eb6b1997711067ffcb445532667ed4a4efc2b334c16421edb52ec5e0324a1c0e570663"
+export const BECH32_ACCOUNT = "cheqd1ehcg0jarxkyxtkzrwcxayedxrskwyftxj4exm9"
+// Testnet RPC nad faucet address
+export const FAUCET_ADDRESS = "cheqd1rnr5jrt4exl0samwj0yegv99jeskl0hsxmcz96"
+export const TESTNET_RPC = "https://rpc.cheqd.network:443"
```
Comment (Eengineer1) [inline tests/constants.ts:6]: Could you move all the above declarations under `src/tests/testutils.test.ts` instead?

I'd probably keep them all in lower case as well.

It's intended to hold all test utils, hence why it's overlooked by test config.

**Review by Eengineer1 — CHANGES_REQUESTED** [review]

File: tests/constants.ts (hunk 2 above)
Comment (Eengineer1) [reply tests/constants.ts:6]: Plus, faucet address is already defined there.

**Review by Eengineer1 — APPROVED** [review]
Approved.

Nice work!

**Pull request merged**